assets/scripts/script.js

```diff
@@ -40,22 +40,18 @@
     }
 
     if (typeof $.fn.lightGallery === 'function') {
-      if ($('.js-gallerya-lightbox').length > 0) {
-        $('.js-gallerya-lightbox').lightGallery({
-          thumbnail: true,
-          showThumbByDefault: false,
-          subHtmlSelectorRelative: true,
-          selector: '.gallerya__image > a'
-        });
-      }
-      if ($('body').hasClass('woocommerce') && $('.images').length > 0) {
-        $('.images').lightGallery({
-          thumbnail: true,
-          showThumbByDefault: false,
-          subHtmlSelectorRelative: true,
-          selector: '.zoom'
-        });
-      }
+      $('.js-gallerya-lightbox').lightGallery({
+        thumbnail: true,
+        showThumbByDefault: false,
+        subHtmlSelectorRelative: true,
+        selector: '.gallerya__image > a'
+      });
+      $('.woocommerce-product-gallery').lightGallery({
+        thumbnail: true,
+        showThumbByDefault: false,
+        subHtmlSelectorRelative: true,
+        selector: '.attachment-shop_single'
```

Comment:
`img[class*="attachment-"]`

Reply:
Applied your suggestion and just pushed a fixup.